Google.Cloud.AspNetCore/Google.Cloud.AspNetCore.DataProtection.Kms/KmsXmlConstants.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2019 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Xml.Linq;
+
+namespace Google.Cloud.AspNetCore.DataProtection.Kms
+{
+    /// <summary>
+    /// Constants used by both <see cref="KmsXmlDecryptor"/> and <see cref="KmsXmlEncryptor"/>.
+    /// </summary>
+    internal static class KmsXmlConstants
```

Comment:
If we have to look at the code in a few months time, I think it will be quicker to understand if we have a sample snippet of XML in comments somewhere in the source.

Reply:
Okay, will do.